### ISSUE TITLE
configure: error: LAPACK and/or BLAS not found

### ISSUE THREAD

**Issue by arpan-das-astrophysics** [amusecode/amuse#825]
Hello, I am trying to install amuse in a remote cluster and I don't have the root privilege to the machine. I followed the instruction given [here](https://amuse.readthedocs.io/en/latest/reference/cartesius.html#running-on-a-cartesius-node). Basically I installed the prerequisites from amuse>doc>install>install.py. However while ./configure I am getting the error configure: error: LAPACK and/or BLAS not found. Any help how to install them without the root access?

**Comment by spzwart** [comment]
Dear Arpan,

It is best if lapac and blas are installed by the sysop. These packages
can be quite hard to install properly and optimally.

Cheers,
Simon

**Comment by rieder** [comment]
I agree with Simon. Usually you should be able to find these or the system admins could install these for you - perhaps they are available as modules already?
Besides that, I don't think configure should give an error on LAPACK and/or BLAS... these are not major requirements for AMUSE (I think only one or two community codes use them). Could you attach the output from configure?

**Comment by arpan-das-astrophysics** [comment]
> I agree with Simon. Usually you should be able to find these or the system admins could install these for you - perhaps they are available as modules already? Besides that, I don't think configure should give an error on LAPACK and/or BLAS... these are not major requirements for AMUSE (I think only one or two community codes use them). Could you attach the output from configure?

Thank you. Yes,
[terminal_output.txt](https://github.com/amusecode/amuse/files/8081121/terminal_output.txt)
 here is the terminal output attached

**Comment by rieder** [comment]
@ipelupessy could this error be due to a recent change? I haven't seen configure fail on this before.

**Comment by ipelupessy** [comment]
yes, this was merged two months ago or so? It currently raises an error (configure.ac) can be converted to a notice
(but then you get fails on the packages that use it (galaxia and new mesa)

**Comment by rieder** [comment]
I think it should be a notice - we also don't fail if some other libraries are not found (e.g. hdf5, required by Simplex)

**Comment by arpan-das-astrophysics** [comment]
Ok I managed to resolve this by creating symbolic links to the BLAS and LAPACK.
